Source/IGListSingleSectionController.m

```diff
@@ -63,9 +63,9 @@ - (instancetype)initWithStoryboardCellIdentifier:(NSString *)identifier
     IGParameterAssert(configureBlock != nil);
     IGParameterAssert(sizeBlock != nil);
     if (self = [super init]) {
-        _identifier = [identifier copy];
```

Comment:
This needs to stay b/c it is coming from consumers of the API. They could pass in an `NSMutableString` and change it later, breaking things.